tmt/steps/__init__.py

```diff
@@ -448,12 +537,13 @@ def wake(self, keys=None):
         """
         if keys is None:
             keys = self._common_keys + self._keys
+        assert keys is not None
```

Comment:
mypy should be able to recognize keys is no longer None, as long as it can inspect self._common_keys and self._keys. Is assert here needed/

Reply:
assert is removed. It was not needed